package_drupal_script.sh

```diff
@@ -138,7 +162,7 @@ if [ -f ${REPO_DIR}/package_additions.sh ]; then
 fi
 
 cd $STAGING_DIR_BASE
-tar -czf drupal_sprint_package.tar.gz $STAGING_DIR_NAME
-zip -9 -r -q drupal_sprint_package.zip $STAGING_DIR_NAME
+tar -czf drupal_sprint_package$QUICKSPRINT_RELEASE.tar.gz $STAGING_DIR_NAME
+zip -9 -r -q drupal_sprint_package$QUICKSPRINT_RELEASE.zip $STAGING_DIR_NAME
 wait
 printf "${GREEN}The sprint tarballs and zipballs are in $(ls $STAGING_DIR_BASE/drupal_sprint_package*).${RESET}\n"
```

Comment:
We don't need to show the whole directory, let's just show the tar/zipballs.

Reply:
@rfay not sure what you mean here?